packages/wrangler/src/deploy/deploy.ts

```diff
@@ -238,11 +238,11 @@ export const validateRoutes = (routes: Route[], assets?: AssetsOptions) => {
 };
 
 export function renderRoute(route: Route): string {
-	let result = "";
+	let result = "https://";
```

Comment:
Unfortunately you will need to accommodate this feature: https://developers.cloudflare.com/workers/configuration/routing/routes/#:~:text=Route%20patterns%20may%20optionally%20begin%20with%20http%3A//%20or%20https%3A//